packages/clay-button/docs/index.js

```diff
@@ -19,14 +19,11 @@ const ButtonDisplayTypesCode = `const Component = () => {
 			<ClayButton displayType="secondary">
 				Button Secondary
 			</ClayButton>
-			<ClayButton displayType={null}>
-				Base Button
-			</ClayButton>
 			<ClayButton displayType="link">
 				Button Link
 			</ClayButton>
-			<ClayButton displayType="unstyled">
-				Button Unstyled
+			<ClayButton displayType="outline-borderless">
```

Comment:
This won't work as expected, you have to add borderless using prop.

```suggestion
			<ClayButton borderless>
```

Reply:
Thanks @matuzalemsteles!
Fixed